download/downloader_test.go

```diff
@@ -1,12 +1,41 @@
 package download
 
 import (
+	"encoding/json"
 	"os"
 	"path"
 	"path/filepath"
 	"testing"
 )
 
+func TestCreateLastUpdateFile(t *testing.T) {
+	tmp := t.TempDir()
+	updateDates := []string{"2021-12-12", "2021-10-20"}
+
+	err := createLastUpdateJSONFile(tmp, updateDates)
+	if err != nil {
+		t.Errorf("Expected no errors on create last_update.txt, got: %v", err)
```

Comment:
1. Estou no meio de uma tentativa de seguir o estilo do Go de ter erros sempre em minúsculas, então acho que aqui poderíamos ter isso aplicado (mesmo sendo um erro de teste)
2. Também colocaria o caminho completo de saída
3. E podemos colocar o erro com _string_



```suggestion
		t.Errorf("Expected no errors on create %s, got: %s", filepath.Join(tmp, "last_update.txt", err)
```